app/views/checkout/payment.scala.html

```diff
@@ -20,6 +38,22 @@
 
             <div class="checkout-container__sidebar">
                 @fragments.checkout.basketPreview(products)
+                <div class="basket-preview__toggle">
```

Comment:
Don't love having BEM fragments `basket-preview__toggle` outside the parent component…

Would rather pass a `showPaymentOptions` boolean to `basketPreview`, or else refactor the markup to make this two separate components.